lib/wasi-tests/tests/wasitests/mod.rs

```diff
@@ -5,26 +5,49 @@
 // The _common module is not autogenerated.  It provides common macros for the wasitests
 #[macro_use]
 mod _common;
-mod close_preopen_fd;
-mod create_dir;
-mod envvar;
-mod fd_allocate;
-mod fd_append;
-mod fd_close;
-mod fd_pread;
-mod fd_read;
-mod fd_sync;
-mod file_metadata;
-mod fs_sandbox_test;
-mod fseek;
-mod hello;
-mod isatty;
-mod mapdir;
-mod path_link;
-mod path_rename;
-mod path_symlink;
-mod poll_oneoff;
-mod quine;
-mod readlink;
-mod wasi_sees_virtual_root;
-mod writing;
+mod snapshot1_close_preopen_fd;
```

Comment:
Perhaps having them in different dirs might be good? (not sure)

Reply:
I agree, I think that's a good change.  It's a bit non-trivial to update it though as it will require updating some of the structure of the code and the way the ignore list works -- I think that'd be a good follow up issue, but I can prioritize it now if you want!